index.js

```diff
@@ -114,6 +114,16 @@ function SendStream(req, path, options) {
     ? normalizeList(opts.extensions, 'extensions option')
     : []
 
+  if (Array.isArray(opts.precompressed)) {
+    this._precompressionFormats = opts.precompressed
+  } else if (opts.precompressed) {
+    this._precompressionFormats = [{encoding: 'br', extension:'.br'}, {encoding:'gzip', extension:'.gz'}]
```

Comment:
Cool, sorry @gmokki, I did not see the link in the linked to comment. Reading through that spec, it sounds like the file framing for the `.br` file format is not the same as the framing used in the `br` Content-Encoding, is that correct? If so, we should not be using that file extension to serve up as if it were the content encoding (I am comparing the content encoding https://tools.ietf.org/html/draft-alakuijala-brotli-11#section-2 to the referenced file framing https://github.com/madler/brotli/blob/master/br-format-v3.txt).

Do you know if we send out a brotli stream that has the file framing around it as `Content-Encoding: br` if the clients will read it? I would imagine that the decoders will choke on the first 4 bytes of the file frame if they are trying to read it as a content stream, no?

If you can try it, can you see what a web browser does when `Content-Encoding` is `br` and the following bytes are the body: `ce b2 cf 81 84 06 00 27`?